apps/web/app/(app)/onboarding/OnboardingNextButton.tsx

```
@@ -0,0 +1,20 @@
+"use client";
+
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import { useSearchParams } from "next/navigation";
+
+export function OnboardingNextButton() {
+  const searchParams = useSearchParams();
+  const currentStep = parseInt(searchParams.get("step") || "1");
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)